packages/next/server/config-shared.ts

```diff
@@ -106,6 +106,9 @@ export interface ExperimentalConfig {
   urlImports?: NonNullable<webpack5.Configuration['experiments']>['buildHttp']
   outputFileTracingRoot?: string
   outputStandalone?: boolean
+  images?: {
+    layoutRaw: boolean
```

Comment:
Nit: why not "rawLayout" (seems more readable?)

Reply:
In my head (and I think @styfle's when he suggested that name) it's from the layout="raw" formulation. I'm open to changing it, or just leaving it since experimental flags don't necessarily need to be designed for discoverability.